Formula/jackett.rb

```diff
@@ -0,0 +1,52 @@
+class Jackett < Formula
+  desc "API Support for your favorite torrent trackers"
+  homepage "https://github.com/Jackett/Jackett"
+  url "https://github.com/Jackett/Jackett/archive/refs/tags/v0.20.805.tar.gz"
+  sha256 "5b37a01517d2230879fdd0b5f5a3344abcc7718adee9584c2b7b40e5b99b721b"
+  license "GPL-2.0-only"
+  head "https://github.com/Jackett/Jackett.git", branch: "master"
+
+  depends_on "dotnet"
+  depends_on macos: :high_sierra
```

Comment:
If it only works on 1/6 supported platforms it might not be a good fit for Homebrew/core

Reply:
Sorry, it is meant to be "High Sierra or later". How can I specify that? I'll try it without any OS-dependency though, just in case.

Reply:
Since homebrew only officially supports Catalina or later you can just drop that line.